set.mm

```diff
@@ -12502,6 +12502,22 @@ the logical conjunction ( ~ df-an ).
 #*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#*#
 $)
 
+  $( Pure implicational calculus is the fragment of propositional logic that
+     uses only material implication, and not negation.  The shortest single
+     axiom for implicational calculus, due to J. Lukasiewicz, appears in
+     Proceedings of the Royal Irish Academy.  Section A:  Mathematical and
+     Physical Sciences , 1948 - 1950, Vol. 52 (1948 - 1950), pp. 25-33.  From
+     this axiom and ~ ax-mp , { ~ peirce , ~ ax-1 , ~ ax-2 } can be proved, and
+     hence all tautologies containing only material implication.  The axiom is
+     proved here starting from { ~ peirce , ~ ax-1 , and ~ ax-2 }.  Therefore,
+     the class of theorems proved by this single axiom, or alternatively {
+     ~ peirce , ~ ax-1 and ~ ax-2 } must be identical.  (Contributed by Larry
+     Lesyna and Jeffrey P. Machado, 18-Jul-2023.) $)
+  impsingle $p |- ( ( ( ph ->
+ps ) -> ch ) -> ( ( ch -> ph ) -> ( th -> ph ) ) ) $=
```

Comment:
I suggest to break up the long expression
`( ( ( ph -> ps ) -> ch )`    ` -> ( ( ch -> ph ) -> ( th -> ph ) ) )`
at the point, where antecedents are separated from the consequent, for better readability and alignment with similar long expressions (see pm2.86 for example).  It further helps to indent the second portion such that it starts at a the same or a later position than the first one.  Only use space characters for indentation, no tabulators.

`From this axiom and ~ ax-mp , { ~ peirce , ~ ax-1 , ~ ax-2 } can be proved`

Do it.  To be accepted as (yet) another axiomatic foundation of propositional logic, the equivalence with the old system must be shown first.  In addition ax-3 should follow from peirce, ax-1, ax-2, by our traditional techniques.

The current content of the PR cannot be accepted, because of the mentioned deficiencies.  But even if they are resolved, there might be a discussion about whether it is important enough to be added in the main portion, and if so, where to add it.

You can instead create your own mathbox, where your idea can live independently of my or anyone else's opinion.